File: app/src/main/java/com/lambdaschool/android_classes/ContactsManager.java
package com.lambdaschool.android_classes;

import java.util.ArrayList;

public class ContactsManager {

    private ArrayList<Contact> contacts;

    public ContactsManager(ArrayList<Contact> contacts) {
        this.contacts = contacts;
    }

    public ContactsManager() {
        this.contacts = new ArrayList<Contact>();
    }

    public ArrayList<Contact> getContacts() {
        return contacts;
    }

    public void createContact(String name, String phone) {
        this.contacts.add(new Contact(name, phone));
    }

    public Contact getLast() {
        return contacts.get(contacts.size() - 1);
    }
}

File: app/src/main/java/com/lambdaschool/android_classes/ContactsManager.kt
package com.lambdaschool.android_classes

class ContactsManager(val contacts: MutableList<Contact> = mutableListOf()) {
    val last: Contact
        get() = contacts[contacts.size - 1]

    fun createContact(name: String, phone: String) {
        this.contacts.add(Contact(name, phone))
    }
}